notebooks/official/automl/sdk_automl_tabular_regression_batch_bq.ipynb

```diff
@@ -0,0 +1,968 @@
+{
```

Comment:
<p>Set the <code>REGION</code> variable, which is used for operations throughout the rest of this notebook.  The following regions supported for Vertex AI. We recommend that you choose the region closest to you. </p><p> </p><p>...</p><p> </p><p>Learn more about <a href="https://cloud.google.com/vertex-ai/docs/general/locations" target="_blank">Vertex AI regions</a><strong>. </strong>- missing period</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>